src/lib.rs

```diff
@@ -25,6 +25,8 @@
 extern crate libc;
 
 pub mod types;
+pub mod object;
```

Comment:
This is a -sys crate dedicated to libglib-2.0, as per [the Cargo convention]. Bindings to libgobject should to go [gobject-sys].

Accordingly, there is no need for glib namespace: all definitions are made at the top level, except the 'intrinsic' types which need to be glob-imported into other crates.

[the Cargo convention]: http://doc.crates.io/build-script.html#*-sys-packages
[gobject-sys]: https://github.com/gi-rust/gobject-sys